Philips.CodeAnalysis.MaintainabilityAnalyzers/Philips.CodeAnalysis.MaintainabilityAnalyzers.csproj

```diff
@@ -28,7 +28,7 @@
     <PackageTags>CSharp MsTest Roslyn CodeAnalysis analyzers Philips</PackageTags>
     <NoPackageAnalysis>true</NoPackageAnalysis>
     <DevelopmentDependency>true</DevelopmentDependency>
-		<Version>1.2.2</Version>
+		<Version>1.2.3</Version>
```

Comment:
I think Edward just checked in a change to claim 1.2.3.

Reply:
Revving it up.